packages/api-server/src/methods/constant.ts

```diff
@@ -0,0 +1,21 @@
+// source: https://github.com/nervosnetwork/godwoken-polyjuice/blob/main/docs/EVM-compatible.md
+export const POLY_MAX_BLOCK_GAS_LIMIT = 12500000;
+export const POLY_MAX_TRANSACTION_GAS_LIMIT = 12500000;
+export const POLY_MIN_GAS_PRICE = 0;
+export const POLY_BLOCK_DIFFICULTY = 2500000000000000;
```

Comment:
```suggestion
export const POLY_BLOCK_DIFFICULTY = BigInt("2500000000000000");
```

Reply:
should I change the other three (POLY_MAX_BLOCK_GAS_LIMIT, POLY_MAX_TRANSACTION_GAS_LIMIT, POLY_MIN_GAS_PRICE) to BigInt too? @classicalliu

Reply:
Seems no need,  `POLY_BLOCK_DIFFICULTY` is too big, I'm not sure it will works fine with number or not